### ISSUE TITLE
Trunk Inventory wont open

### ISSUE THREAD

**Issue by twigmasters** [dutchplayers/FiveM-Resources#81]
**Describe the bug**
Followed the readme files to install step by step. When trying to access the trunk inventory pressing F3 when standing behind the vehicle I can see the trunk like it tries to open but it locked. If I open the trunk using /trunk or using the vehicle control menu the trunk will open. But when I press F3 to access it with the trunk open it closes the trunk.

**To Reproduce**
1. Standing behind the vehicle and pressing F3
2. Trunk acts like it is locked
3. Open the trunk using a menu or /trunk command
4. Pressing F3 closes the trunk
5. Pressing F2 while the trunk is open shows player inv but "Secondary Inventory not available"

**Expected behavior**
I expected to be able to access the trunk inventory

**ESX version:**
 - Version 1.1 (newest version breaks lot of scripts currently on my server)

**Followed Wiki**
Yes to the letter

Note: Player inventory seems to work as expected.

**Comment by FiveEYZ** [comment]
think its because of this inside `esx_inventoryhud_trunk\client\esx_trunk-cl.lua` line 180
```
Citizen.CreateThread(function()
    while true do
        Wait(0)
        local pos = GetEntityCoords(GetPlayerPed(-1))
        if CloseToVehicle then
            local vehicle = GetClosestVehicle(pos["x"], pos["y"], pos["z"], 2.0, 0, 70)
            if DoesEntityExist(vehicle) then
                CloseToVehicle = true
            else
                CloseToVehicle = false
                lastOpen = false
                ESX.UI.Menu.CloseAll()
                SetVehicleDoorShut(lastVehicle, 5, false)
            end
        end
    end
end)
```

**Comment by FiveEYZ** [comment]
ok, a quick fix would be to change:
`local vehicle = GetClosestVehicle(pos["x"], pos["y"], pos["z"], 2.0, 0, 70)`
with:
`local vehicle = VehicleInFront()`

then the trunk will remain open until u start to move/look away from the vehicle

**Comment by twigmasters** [comment]
with this edit I can open the trunk with F3 but still not access inventory in the trunk. The regular personal inventory hud is working fine but still seeing "secondary inventory is not available" when standing at the trunk with it open and pressing F2. 

I also noticed this in the F8 consol window

![esx_inventory_trunk error](https://user-images.githubusercontent.com/64451461/81463677-42791500-9189-11ea-8e7e-6b07776b2c39.png)

**Comment by FiveEYZ** [comment]
im sure updating to latest ESX 1.2 will fix the remaining errors

**Comment by twigmasters** [comment]
Instead of doing a complete fresh install of my server which would take hours, I disabled all my resources not needed by the inventory system and updated to ESX 1.2. This of course breaks a bunch of my resources which is another reason I disabled them.

I am still getting the error above in the F8 console when trying to access the trunk inventory. Regular player inventory is working normally. 

Here is a list of the resources I left enabled
vmenu
mysql-async
async
es_extended
esx_menu_default
esx_menu_dialog
z-loadingscreen
esx_datastore
esx_addonaccount
cron
b1g_notify
esx_addoninventory
esx_inventoryhud
esx_inventory_trunk

I even cleared the cache to be sure and same error.

**Comment by FiveEYZ** [comment]
there is also https://github.com/extendedmode/extendedmode that might work, cant promise

**Comment by FiveEYZ** [comment]
so the error seems to be when it tried to get account data from the database, do u have a column in users called `accounts`?

**Comment by twigmasters** [comment]
Yes there is a column for accounts, I just checked. 
The link for extendedmode.... is that a replacement for 1.2 ESX? or an added resource. I am looking at it to see if it could help.

I have essentialmode not sure if that's the same thing but it is disabled because ESX1.2 turns it off anyway.

**Comment by twigmasters** [comment]
Well after having to edit the database I was able to get in the server with extendedmode replacing ESX1.2. Well I am still getting the same error in the F8 console again. 

![esx_inventory_trunk error](https://user-images.githubusercontent.com/64451461/81487481-28900e80-922b-11ea-89ae-0fd8fac1894b.png)

I guess all I can do now is start from scratch which could take hours. There has to be an easier solution, I have been fighting with this for over a week!

**Comment by FiveEYZ** [comment]
sorry i have no idea, have never encountered this myself

guess u could always debug and backtrack those functions that gives error, see what data they ask for and see what data they get

**Comment by twigmasters** [comment]
I have done a bunch of work trying to narrow down what is giving the error. I even started putting a new server structure together to try and weed out what the issue could be. Long story short, on a whim I downloaded the pack again to replace what I had. I installed them and to my surprise it started working!

Only issue I am having now is that part of the menu is still in dutch even after changing the [nl} to [en} in the configs.

![inv language](https://user-images.githubusercontent.com/64451461/81516238-3914b780-9305-11ea-892f-483993518a2b.png)

So in the end I have no clue what fixed it but its working for the moment. Thanks for all your help now I just need to figure this menu issue out.
